packages/create-strapi-starter/create-strapi-starter.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const commander = require('commander');
+const packageJson = require('./package.json');
+const buildStarter = require('./utils/build-starter');
+
+const program = new commander.Command(packageJson.name);
+
+let projectName;
+
+program
+  .version(packageJson.version)
+  .arguments('<directory>')
+  .arguments('<starterurl>')
+  .option('--no-run', 'Do not start the application after it is created')
```

Comment:
can we just remove all the options that should be passed down & just send them as is to the generateNew ? 
Just trying to avoid redeclaring all of the options